setup.py

```diff
@@ -4,7 +4,7 @@
 requirements = {
     'install': [
         'chainer>=5.0.0',
-        'onnx>=1.4.0,<1.5',
+        'onnx>=1.5.0,<1.6',
```

Comment:
I'd like to support onnx 1.4.x because some runtimes are not support 1.5 or partially support. For your information, onnxruntime==0.4.0 is run with onnx 1.4.x